compiler/src/main/kotlin/edu/cornell/cs/apl/viaduct/codegeneration/CodeGeneratorContext.kt

```diff
@@ -22,9 +22,13 @@ interface CodeGeneratorContext {
     // returns a fresh kotlin name for baseName
     fun newTemporary(baseName: String): String
 
+    fun tempKotlinNameToProtocol(kotlinName: String): Protocol
```

Comment:
What is this function for? I think we said this should be private to each `CodeGenerator` if the need it.

Reply:
This is also ABY specific, will remove from this PR